packages/container/tests/container_test.js

```diff
@@ -724,15 +725,16 @@ if (EMBER_MODULE_UNIFICATION) {
     ['@test The container can pass a namespace to lookup']() {
       let PrivateComponent = factory();
       let type = 'component';
-      let namespace = 'my-addon';
+      let targetNamespace = 'my-addon';
       let name = 'my-component';
-      let rawString = `${namespace}::${name}`;
+      let rawString = `${targetNamespace}::${name}`;
+      let specifier = `${type}:${name}`;
       let resolver = new ModuleBasedTestResolver();
       let registry = new Registry({resolver});
 
       resolver.add({
-        specifier: type,
-        rawString: rawString
+        specifier,
+        targetNamespace
```

Comment:
This could just be `namespace`, I don't think `target` is useful to discern it from anything else?